server/services/project_search_service.py

```diff
@@ -1,12 +1,21 @@
 import geojson
-
 from shapely.geometry import Polygon, box
-from server.models.dtos.project_dto import ProjectSearchDTO, ProjectSearchResultsDTO, ProjectSearchResultDTO, Pagination, ProjectSearchBBoxDTO
+from server.models.dtos.project_dto import ProjectSearchDTO, ProjectSearchResultsDTO, ProjectSearchResultDTO, \
+    Pagination, ProjectSearchBBoxDTO
 from server.models.postgis.project import Project, ProjectInfo
 from server.models.postgis.statuses import ProjectStatus, MappingLevel, MappingTypes, ProjectPriority
-from server.models.postgis.utils import NotFound, ST_Intersects, ST_MakeEnvelope, ST_Transform
+from server.models.postgis.utils import NotFound, ST_Intersects, ST_MakeEnvelope, ST_Transform, ST_Area
 from server import db
 from flask import current_app
+from geoalchemy2 import shape
+import math
+from server.services.users.user_service import UserService
+
+# max area allowed for passed in bbox
+# 243375 is arbitrarily chosen map maximum map width in meters
+# multiply by 1.5 to give a buffer beyond edge of map
+# raise to power 2 to get the maxim allowed area
+MAX_AREA = math.pow(243375 * 1.5, 2)
```

Comment:
Is it not simpler to just just set the MAX area to the result of that calculation??